README.md

```
@@ -1,3 +1,513 @@
+
+# Logging
+logging:
+  level: INFO
+  file: ~/.local/share/cortex/cortex.log
+```
+
+---
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)